### PR TITLE
Replace raw url on thank-you page

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This replaces the raw URL as a link text on the thank-you page template. Previously, it didn't provide enough context for screen readers  and made navigation difficult without the surrounding information. Translations also had to be updated.

### How to review
Load a schema with thank-you page and the link enabled and visible (there is extra template logic for when the link is displayed). Check if the new link text is displayed.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: templates/thank-you.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <p class="ons-u-mb-s">{{ _("Your answers will be processed in the next few weeks.") }}
       {{ _("We may contact you to query your answers via phone or secure message.") }}</p>
     <p class="ons-u-mb-s">{{ _("For more information on how we use this data.") }}<br>
-      <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
+      <a href="https://www.ons.gov.uk/surveys">{{ _("Learn more about how we use this data") }}</a></p>
```
Comment (berroar) [inline templates/thank-you.html:48]: Isn't the text above supposed to change to match the text on the ticket?

Comment (petechd) [reply templates/thank-you.html:48]: Yeah, description was slightly confusing, added now, thanks.

**Review by rmccar — APPROVED** [review]

**Review by VirajP1002 — APPROVED** [review]
Test changes pass and the html changes look good to me - I had a look in a few schemas and the changes look good 👍

File: templates/thank-you.html (hunk 1 above)
Comment (petechd) [reply templates/thank-you.html]: I confirmed with Donna this should be the only text above the link.

**Review by berroar — APPROVED** [review]

**Pull request merged**